### ISSUE TITLE
Remove @qsi imports

### ISSUE THREAD

**Issue by AaronBuxbaum** [4Catalyzer/cli#323]
https://github.com/4Catalyzer/cli/blob/c0b4d76ad1485da5a95a695b12375959ec4d3350/packages/pedantic/tests/fixtures/imports.js#L7

We're moving on to private GitHub Packages imports, so I'm not sure if these imports will work long-term. They're kind of questionable to begin with in a pseudo-circular import manner

**Comment by jquense** [comment]
I don't think this fixture resolves to anything, it's there to test linting I believe

**Comment by AaronBuxbaum** [comment]
Ah hm... wouldn't it be better to make it clear that they're not real imports, then? Something like `@fake` removes (or at least reduces) needing to know that
